### DISCUSSION TITLE
The neck posture is disturbed by the mouth

### DISCUSSION THREAD

**Discussion by 43998213** [yinguobing/head-pose-estimation#44] [Q&A]
The neck posture is disturbed by the mouth

**Comment by 43998213** [comment]
include https://github.com/natanielruiz/deep-head-pose, The posture will shake,When you open your mouth.

**Comment by yinguobing** [comment]
Yes. The pose is estimated by solving a PnP problem that takes all face landmarks into consideration. It will be affected by relative changing distributions of some marks. Ignoring these marks may be helpful.

**Comment by 43998213** [comment]
> Yes. The pose is estimated by solving a PnP problem that takes all face landmarks into consideration. It will be affected by relative changing distributions of some marks. Ignoring these marks may be helpful.

I think the problem come from res10_300x300_ssd_iter_140000.caffemodel,When you open your mouth, the identification box will become larger and the point will become larger.

**Comment by 43998213** [comment]
god job I learn a lot and thanks a million

**Comment by yinguobing** [comment]
Glad to hear that!
